src/CodeGenerator/AutoloadGenerator.php

```diff
@@ -7,87 +7,93 @@
 
 namespace Zend\Di\CodeGenerator;
 
-use Zend\Code\Generator\DocBlockGenerator;
-use Zend\Code\Generator\FileGenerator;
-use Zend\Code\Generator\ClassGenerator;
-use Zend\Code\Generator\MethodGenerator;
-use Zend\Code\Generator\PropertyGenerator;
-use Zend\Code\Generator\PropertyValueGenerator;
+use SplFileObject;
+use Throwable;
+use Zend\Di\Exception\GenerateCodeException;
+use function array_keys;
+use function array_map;
+use function file_get_contents;
+use function implode;
+use function str_repeat;
+use function strtr;
+use function var_export;
 
 class AutoloadGenerator
 {
     use GeneratorTrait;
 
-    private $namespace;
+    const CLASS_TEMPLATE = __DIR__ . '/../../templates/autoloader-class.template';
+    const FILE_TEMPLATE = __DIR__ . '/../../templates/autoloader-file.template';
```

Comment:
Please add visibility modifiers to class constants. (You get to choose which visibility makes sense.)

Reply:
Sure. I thought this was only available in 7.2 and up. My mistake.